### ISSUE TITLE
Value Error: The following variables were created within a Lambda layer (anchors) but are not tracked by said layer

### ISSUE THREAD

**Issue by fizaict** [nageshsinghc4/Brain-Tumor-Detection-using-Mask-R-CNN#2]
I am getting the above error.  

Keras Version: 2.4.3
Tensorflow Version: 2.3.0

Can you please tell me what should be the version of Keras and TensorFlow so that this code will run successfully?

**Comment by fizaict** [comment]
@nageshsinghc4 still waiting

**Comment by Deorbiting** [comment]
I've tried changing the version but to no avail, I was on the same versions of TF and Keras as yourself.

I changed to these versions:
Keras Version: 2.1.5
Tensorflow Version: 1.15

Hopefully We can expect some sort of update soon
